app/data/census_individual.json

```diff
@@ -1922,15 +1940,12 @@
                                                     "value": "Irish"
                                                 },
                                                 {
-                                                    "label": "None",
-                                                    "value": "None"
+                                                    "label": "Other",
+                                                    "value": "Other"
                                                 },
                                                 {
-                                                    "label": "Other (please specify)",
-                                                    "value": "Other",
-                                                    "other": {
-                                                        "label": "Please specify other passports held"
-                                                    }
+                                                    "label": "None",
+                                                    "value": "None"
```

Comment:
See https://github.com/ONSdigital/eq-survey-runner/pull/691#issuecomment-266376207 I think these need to be swapped so Other is last.